src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -344,6 +360,35 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
             if (controllerData.xrController) {
                 controllerGlobalPosition = controllerData.xrController.pointer.position;
                 controllerData.xrController.getWorldPointerRayToRef(controllerData.tmpRay);
+                const hand = controllerData.xrController.inputSource.hand;
+                if(hand)
+                {
+                    const xrJoint = hand.get("index-finger-tip");
+                    if(xrJoint)
+                    {
+                        let pose = _xrFrame.getJointPose!(xrJoint, this._xrSessionManager.referenceSpace);
+                        if (!pose || !pose.transform) {
+                            return;

```

Comment:
Probably shouldn't return here. We can still do far interaction processing with no joint data.

Reply:
good catch, i was thinking about it and slipped my mind